.azure-pipelines/hpu-tests.yml

```
@@ -33,7 +33,9 @@ jobs:
       displayName: 'Instance HW info'
 
     - bash: |
+        pip install . --requirement requirements/extra.txt
         pip install . --requirement requirements/test.txt
+        pip uninstall -y horovod  # HotFix: not sure how it got there...
```

nit: prefer TODO since it's what I usually grep for

```suggestion
        pip uninstall -y horovod  # TODO: not sure how it got there...
```